### ISSUE TITLE
Add getBalances

### ISSUE THREAD

**Issue by jishankai** [QuarkChain/quarkchain-web3.js#25]
Now, web3.qkc.getBalance only returns balance of QKC. Will we add getBalances to support native tokens as JRPC does?

**Comment by qizhou** [comment]
Yes, I think so.  +Junjia He <junjiah@quarkchain.org> what do you think?

- Qi
